apps/2i/hotel/frontend/src/app/admin/_Components/DetailsImage.tsx

```
@@ -0,0 +1,72 @@
+import { Button } from '@/components/ui/button';
+import { Card, CardContent, CardHeader, CardTitle } from '@/components/ui/card';
```

ESLint

[error] 'CardHeader' is defined but never used.

(no-unused-vars)